libraries/Ethernet/src/Ethernet.cpp

```diff
@@ -56,6 +62,11 @@ int CEthernet::begin(IPAddress local_ip, IPAddress dns_server, IPAddress gateway
 int CEthernet::begin(IPAddress local_ip, IPAddress dns_server, IPAddress gateway, IPAddress subnet) {
 /* -------------------------------------------------------------------------- */  
   
+  this -> ethernetTimer = new EthernetClock();
```

Comment:
I'd surround the clock initialization (and deinitialization) between `#ifdef ETHERNET_CLK_PIN` to take into account possible future platform that use a different method to provide the 25Mhz clock